akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -157,9 +144,27 @@ object Behaviors {
    * need and in fact should not use (close over) mutable variables, but instead
    * return a potentially different behavior encapsulating any state changes.
    */
-  def immutable[T](onMessage: (ActorContext[T], T) ⇒ Behavior[T]): Immutable[T] =
+  def receive[T](onMessage: (ActorContext[T], T) ⇒ Behavior[T]): Immutable[T] =
     new Immutable(onMessage)
 
+  /**
+   * Simplified version of [[receive]] with only a single argument - the message
+   * to be handled. Useful for when the context is already accessible by other means,
+   * like being wrapped in an [[setup]] or similar.
+   *
+   * Construct an actor behavior that can react to both incoming messages and
+   * lifecycle signals. After spawning this actor from another actor (or as the
+   * guardian of an [[akka.actor.typed.ActorSystem]]) it will be executed within an
+   * [[ActorContext]] that allows access to the system, spawning and watching
+   * other actors, etc.
+   *
+   * This constructor is called immutable because the behavior instance does not
+   * need and in fact should not use (close over) mutable variables, but instead
+   * return a potentially different behavior encapsulating any state changes.
+   */
+  def receiveMessage[T](onMessage: T ⇒ Behavior[T]): Immutable[T] =
```

Comment:
is it confusing to return something named `Immutable` now? what could we call it instead? `MessageHandler` or `BehaviorBuilder`?

Reply:
Yeah, was struck by this as well... Not sure; I guess some alternative name would be good hm... Not so happy with builder since at the same time it is a Behavior already hm

Reply:
Maybe something short like `Receive[]` or that conflicts with things too much? (exists in abstract actor)

Reply:
Seems it works out well in both DSLs